thorlcr/activities/thdiskbase.cpp

```diff
@@ -131,7 +131,10 @@ void CDiskReadMasterBase::getActivityStats(IStatisticGatherer & stats)
 
 void CDiskReadMasterBase::done()
 {
-    diskAccessCost = calcFileReadCostStats(true);
+    // Publish file properties for regular disk read activities (exclude spill reads)
+    // future: consider tracking and recording spill file costs
+    bool publishFileProps = (TAKspillread != container.getKind());
+    diskAccessCost = calcFileReadCostStats(publishFileProps);
```

Comment:
this should really ascertain whether the plane this is writing to has any cost..
spill plane *could* be on a plane that has read cost.

But same would be true of writing temps if so..

What happens if instead of checking if temp, calcFileAccessCost is called but returns 0 (because [spill] plane has no associated cost) and the determination of whether to publish or not is based on whether there's any calculated cost produced instead?